src/utils/css-attr-is-valid.ts

```diff
@@ -0,0 +1,25 @@
+export const cssAttrIsValid = (attr: string, value: unknown) => {
+  const iframe = document.createElement('iframe')
+  Object.apply(iframe.style, {
+    display: 'none',
+    // in case `display: none` not supported
+    width: '0px',
+    height: '0px',
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } as any)
+  document.body.appendChild(iframe)
+  if (!iframe.contentDocument) {
+    return true
+  }
+  const style = iframe.contentDocument.createElement('style')
+  style.innerHTML = `.test-css { ${attr}: ${value}; }`
+  iframe.contentDocument.head.appendChild(style)
+  const { sheet } = style
+  if (!sheet) {
+    return true
+  }
+  const result = sheet.cssRules[0].cssText
+  iframe.remove()
+  const match = result.match(new RegExp(`${attr}:`))
+  return !!match
+}
```

Comment:
What if instead of the iframe, we use CSS supports?

```
CSS.supports(propertyName, propertyValue)

CSS.supports('color','red') 
//True.
CSS.supports('color', '#007')
//True. 

CSS.supports('color', 'random')
//False. 
CSS.supports('colours', 'red')
//False. 
```

Reply:
I'm not sure if this will work as we want, but it's just a suggestion

Reply:
This seems like exactly what I was trying to achieve. You can try on the browser console:

```js
> CSS.supports('border', '1px solid red')
< true
> CSS.supports('border', '1px solid redd')
< false
```

Will update later.